### ISSUE TITLE
Advenced Settings from Project Configuration not showing up

### ISSUE THREAD

**Issue by researchdeveloping** [aws/lumberyard#484]
**Describe the bug**
The Advanced Settings from Project Configuration window is not showing up

**Steps to reproduce**
1. Run Project Configuration,
2. Run Advanced Settings.

**Expected behavior**
Open Advanced Settings window.

**Lumberyard version**
1.23

**Comment by AMZN-Gene** [comment]
Hi @researchdeveloping!  I wasn't able to reproduce this problem.  Sometimes Advanced Settings might fail to open if the project build is out of sync with the enabled gems.  Could you try recompiling and let me know if Advance Settings shows up?

**Comment by wcbx** [comment]
This is a known issue. https://forums.awsgametech.com/t/project-configurator-advanced-settings-and-lyzard-exe-crash/7846/19

@AMZN-Gene it's not clear why some of you aren't reproducing it while a very significant portion of users are. I've did clean installs from 1.23.1 and updated all the executable using git_bootstrap and still lyzard crashes (which causes advanced settings not to show up). I've also did clean recompiles from default *and* empty templates. This is certainly a bug and not a user error.

**Comment by AMZN-Gene** [comment]
Thanks for also linking the forum post as well.  I'm going to try reproducing this from a fresh pc; it's possible my dev environment is saving me somehow.  Sorry this has been happening all the way back since 1.21 for you :c  Stay tuned!

**Comment by researchdeveloping** [comment]
Hi @AMZN-Gene I just reinstaled LY 1.23.1 from windows installer and rebuild all - unfortunatelly problem still exist on my side.

**Comment by AMZN-Gene** [comment]
Ah, so you're seeing the problem, not only with Git, but even when using the installer here: https://aws.amazon.com/lumberyard/downloads/ ?

**Comment by researchdeveloping** [comment]
Yes, @AMZN-Gene the problem exists for git and windows installer versions.

**Comment by AMZN-Gene** [comment]
So sorry to hear this is happening.  If you open cmd inside dev\Bin64vc141\
and run Lyzard.exe, do you get any useful output?  
I noticed others mentioning Perforce in the forums, while I do get warnings about Perforce setup, the app still manages to stay alive.

**Comment by AMZN-Gene** [comment]
Going off the Perforce warning, I tried again after wiping Perforce and I can now reproduce the issue!  Escalating this now to the Lyzard team... stay tuned!

**Comment by CyberNoble** [comment]
Hello Amazon Team,i have this problem too.All other func are showin up but advanced settings -no.Win10.It's start since first launch i can't do it on default samples too.And i just recomp but still have this problem.

**Comment by AMZN-Gene** [comment]
@CyberNoble thanks for the information.  The crash seems tied to a problem with error/warning handling.  People will typically see that crash when not having Perforce set up, not because of any Perforce issue, but rather the messaging system itself leads to the crash.  In the meantime, if you're using the Advanced Settings to edit the System Entity properties a workaround might be to edit the desired values via C++ directly.  Otherwise, if you'd like to edit the Memory Settings, you can do so by hand editing dev\<your_project>\Config\Game.xml. 
Sorry for the inconvenience, the team is now actively working on a fix.

**Comment by researchdeveloping** [comment]
Hello @AMZN-Gene, I am sorry for late answer, I just miss notification but as I see you get it and working on it :) Thank You! I am awaiting fig in upcoming release :)

**Comment by AMZN-Gene** [comment]
Absolutely!  To be clear, it's likely this fix won't make it into the next release.  Was editing the config XMLs a valid workaround in the short term?

**Comment by wcbx** [comment]
This is fixed for me in 1.26.0.0
